src/bitcoin-cli.cpp

```diff
@@ -398,10 +406,12 @@ class NetinfoRequestHandler : public BaseRequestHandler
                 const double min_ping{peer["minping"].isNull() ? -1 : peer["minping"].get_real()};
                 const double ping{peer["pingtime"].isNull() ? -1 : peer["pingtime"].get_real()};
                 const std::string addr{peer["addr"].get_str()};
+                const std::string age{conn_time == 0 ? "" : ToString((m_time_now - conn_time) / 60)};
                 const std::string sub_version{peer["subver"].get_str()};
-                m_peers.push_back({peer_id, mapped_as, version, conn_time, last_blck, last_recv, last_send, last_trxn, min_ping, ping, addr, network, sub_version, is_block_relay, is_outbound});
+                m_peers.push_back({peer_id, mapped_as, version, last_blck, last_recv, last_send, last_trxn, min_ping, ping, addr, age, network, sub_version, is_block_relay, is_outbound});
                 m_max_id_length = std::max(ToString(peer_id).length(), m_max_id_length);
                 m_max_addr_length = std::max(addr.length() + 1, m_max_addr_length);
+                m_max_age_length = std::max(ToString(age).length(), m_max_age_length);
```

Comment:
Hm, age is already a string.
```suggestion
                m_max_age_length = std::max(age.length(), m_max_age_length);
```

Reply:
fixed